python/ray/scripts/scripts.py

```diff
@@ -1926,6 +1927,93 @@ def cpp(show_library_path, generate_bazel_project_template_to, log_style,
                 " && bash run.sh"))
 
 
+@cli.group(hidden=True)
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.pass_context
+def job(ctx, address: Optional[str]):
+    if address is None:
+        if "RAY_ADDRESS" not in os.environ:
+            raise ValueError(
+                "Address must be specified using either the --address flag "
+                "or RAY_ADDRESS environment variable.")
+        address = os.environ["RAY_ADDRESS"]
+
+    ctx.obj = JobSubmissionClient(address)
```

Comment:
Because the `JobSubmissionClient` is being initialized here, users can't view the help documentation for the sub commands unless they provide a valid address. Should we instead put this logic in the sub commands directly?

Reply:
Ah that's really annoying. I can duplicate it to all of the subcommands instead then!